pkg/network/node/networkpolicy.go

```diff
@@ -42,6 +42,8 @@ type networkPolicyPlugin struct {
 	namespaces map[uint32]*npNamespace
 	// nsMatchCache caches matches for namespaceSelectors; see selectNamespaceInternal
 	nsMatchCache map[string]*npCacheEntry
+	// podMatchCache caches matches for podSelectors; see selectPodsInternal
+	podMatchCache map[string]map[string]*npPodCacheEntry
```

Comment:
Maybe rename `npCacheEntry` to `npNamespaceCacheEntry` or `npNSCacheEntry` now. (As a separate commit before this one)